### ISSUE TITLE
how to add new loss in assignment2 pytorch nootbook?

### ISSUE THREAD

**Issue by Cb1ock** [ccmoony/Stanford-CS231n#1]
在这个作业的最后一部分要求实现一个大于70正确率的网络，并且随便你用什么网络，loss，optmizer，等等。但是我不知道如何使用其他的loss。

**Comment by ccmoony** [comment]
softmax 和cross-entropy loss 还有其他的都可以试试。我用的是题目中给的train_part34函数（具体实现在前面的cell中），默认用的是cross-entropy loss，你可以通过修改train_part34用其他损失函数看会不会有更好的accuracy。
